### ISSUE TITLE
Project doesn't build in Android Studio 3.1

### ISSUE THREAD

**Issue by Mura75** [yandex/mapkit-android-demo#2]
Thank you for new version of library.
Project doesn't build in Android Studio 3.1 and show Java compiler error:

>Caused by: com.android.builder.dexing.DexArchiveBuilderException: Failed to process /Users/.gradle/caches/transforms-1/files-1.1/runtime-3.0.0.aar/84cd871fb4c5b7f20232a4b809b354bd/jars/classes.jar
Caused by: com.android.builder.dexing.DexArchiveBuilderException: Error while dexing.
Caused by: java.lang.ArrayIndexOutOfBoundsException
Caused by: java.lang.ArrayIndexOutOfBoundsException: -2147483648

**Comment by IUdalov** [comment]
Fixed with Android Studio 3.1.2 🎉.

**Comment by ilyamodder** [comment]
No, it doesnt.

**Comment by IUdalov** [comment]
Which AS version?

**Comment by ilyamodder** [comment]
3.1.2. Gradle 4.4, Android Gradle plugin 3.1.2

**Comment by debian17** [comment]
Android Studio 3.1.3, Android Gradle plugin 3.1.3, Gradle 4.4, same problem

**Comment by ilyamodder** [comment]
Found a workaround, just add it to gradle.properties file:
```
android.enableD8=false
```

**Comment by bsodKeks** [comment]
кто по русски общается? объясните как построить в итоге? всю голову сломал

**Comment by onchukovdenis** [comment]
@bsodKeks, добавь в файл gradle.properties строчку android.enableD8=false

**Comment by josdem** [comment]
```properties
android.enableD8=false
```
Is deprecated and should not be used anymore.

**Comment by markizdeviler** [comment]
Today AS 3.2 version released in stable channel. To my point of view DX totally replaced with D8. 
```gradle.properties
android.enableD8=false
``` 
This trick shouldn't work with new version of AS. https://issuetracker.google.com/issues/77240639 says The fix should be in Android Studio 3.2 -- unfortunately it didn't make it for Android Studio 3.1.

Some one knows, Is there bug fixed in AS 3.2?

**Comment by vbazhenov** [comment]
Everything seems to work just fine with Android Studio 3.2. Can anyone else confirm?
